styles.css

```diff
@@ -0,0 +1,567 @@
+  padding-bottom: 80px;
+  font-size: 18px;
+  color: rgb(49, 49, 49);
+}
+
+.main-program {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: rgb(29, 27, 27);
+  background-image: url(./img/rows.png);
+  background-repeat: repeat;
+  padding: 3%;
+  margin: 1%;
+}
+
+h2 {
+  color: white;
+  font-size: 24px;
+}
+
+hr {
+  width: 100%;
+  border: 1px solid rgb(236, 50, 50);
+  margin: 3% 0 10% 0;
+}
+
+.Program {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgba(82, 81, 81, 0.6);
+  gap: 15px;
+}
+
+.Program h3 {
+  color: rgb(255, 73, 37);
+  font-size: 25px;
+  padding: 0;
+}
+
+.Program p {
+  font-size: 13px;
+  color: white;
+  padding: 1%;
+  padding-left: 5px;
+  width: 50%;
+  margin: 1%;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.program-pics {
+  height: 60px;
+}
+
+.program-pics1 {
+  width: 50px;
+  padding-left: 9px;
+}
+
+.program-pics2 {
+  width: 50px;
+  padding-left: 10px;
+}
+
+button {
+  /* background-color:   rgb(218, 69, 40); */
+  background-color: transparent;
+  border: 3px solid #ff4848;
+  color: white;
+  width: 300px;
+  height: 50px;
+  padding: 4px 8px;
+  margin: 4%;
+  font-size: 15px;
+  text-transform: uppercase;
+  cursor: pointer;
+  display: inline-block;
+  transition: 0.3s;
+}
+
+button:hover {
+  animation: pulse 1s infinite;
+  transition: 0.3s;
+}
+
+@keyframes pulse {
+  0% {
+    transform: scale(1);
+  }
+
+  70% {
+    transform: scale(0.9);
+  }
+
+  100% {
+    transform: scale(1);
+  }
+}
+
+.featured-speakers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-title h2 {
+  color: #ec5242;
+  padding-top: 20px;
+  font-size: 34px;
+}
+
+.container1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3%;
+  margin: 1%;
+  padding-top: 30px;
+  gap: 10px;
+}
+
+.speakers-grid {
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
+  gap: 10px;
+}
+
+.speaker-desc {
+  display: flex;
+  flex-direction: column;
+}
+
+.speaker-image {
+  width: 120px;
+  height: 120px;
+}
+
+.speaker-desc h5 {
+  font-size: 25px;
+  color: rgb(92, 88, 88);
+}
+
+.speaker-desc h6 {
+  font-size: 13px;
+  color: #ec5242;
+}
+
+.speaker-desc hr {
+  width: 15%;
+  border: 1px solid rgb(180, 177, 177);
+  margin-bottom: 10px;
+}
+
+.speaker-desc p {
+  font-size: 11px;
+  width: auto;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-color: #434344;
+  margin-top: 5%;
+  align-items: center;
+  justify-content: center;
+}
+
+.companies {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+  margin: 2% 0;
+  padding: 5%;
+}
+
+.brands .fa {
+  font-size: 33px;
+  color: #a09595;
+  text-align: center;
+}
+
+.trademark {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: white;
+}
```

Comment:
- [ ] @Hisoka37 great work implementing the changes requested however you forgot to change the space between the logo and the copyright in your home page, you only did it in the about page.